tests/test_figure_resampler.py

```diff
@@ -924,6 +942,60 @@ def test_fr_object_binary_data():
     fig.add_trace({"name": "s0"}, hf_y=binary_series_o, limit_to_view=True)
     assert binary_series_o.dtype == object
     assert len(fig.hf_data) == 1
-    assert fig.hf_data[0]["y"].dtype == "int64"
-    assert fig.data[0]["y"].dtype == "int64"
+    assert (fig.hf_data[0]["y"].dtype == "int32") or (fig.hf_data[0]["y"].dtype == "int64")
+    assert str(fig.data[0]["y"].dtype).startswith("int")
     assert np.all(fig.data[0]["y"] == binary_series)
+
+
+def test_fr_copy_grid():
```

Comment:
why are there no tests / checks for _grid_str?

Reply:
Valid remark! Added these in the latest commit :)